### PR TITLE
[Android/Test] img classification using NNFW

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add simple testcase of image classification result using NNFW.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSingleShot.java [hunk 1]
```diff
@@ -200,7 +192,41 @@ public void testClassificationResult() {
...
+
+    @Test
+    public void testClassificationResultNNFW() {
+        if (!NNStreamer.isAvailable(NNStreamer.NNFWType.NNFW)) {
+            /* cannot run the test */
+            return;
+        }
+
+        try {
+            SingleShot single = new SingleShot(APITestCommon.getTFLiteImgModel(),
```
Comment (gichan-jang) [inline api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSingleShot.java:226]: Don't need to check MANIFEST file?

Comment (jaeyun-jung) [reply api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSingleShot.java:226]: Right, updated.

**Review by gichan-jang — APPROVED** [review]

**Review by anyj0527 — APPROVED** [review]
LGTM 👍🏼👍🏼

**Comment by jaeyun-jung** [comment]
> Check if you don't need to add negative cases here.

@myungjoo this testcase is simply for validating inference result of nn-runtime using tf-lite model.

**Pull request merged**